pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -192,6 +192,21 @@ func TestClusterDeploymentReconcile(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "Check firstTimeInstallAnnotation in install job",
+			existing: []runtime.Object{
+				testClusterDeployment(),
+				testSecret(corev1.SecretTypeDockerConfigJson, pullSecretSecret, corev1.DockerConfigJsonKey, "{}"),
+				testSecret(corev1.SecretTypeOpaque, sshKeySecret, adminSSHKeySecretKey, "fakesshkey"),
+			},
+			validate: func(c client.Client, t *testing.T) {
+				cd := getCD(c)
+				if value := cd.Annotations[firstTimeInstallAnnotation]; value != "true" {
```

Comment:
I would just add this check to the test for creating an install job 
https://github.com/openshift/hive/pull/442/files#diff-ae597ba7bf52a3a731829cf7f676741bR150

Reply:
Added the check to https://github.com/openshift/hive/blob/master/pkg/controller/clusterdeployment/clusterdeployment_controller_test.go#L124-L136 as it should be part of default install job. Also https://github.com/openshift/hive/blob/master/pkg/controller/clusterdeployment/clusterdeployment_controller_test.go#L138 does not have the annotation as it has https://github.com/openshift/hive/blob/master/pkg/controller/clusterdeployment/clusterdeployment_controller_test.go#L141